### PR TITLE
Features/buyproduct

### PR DESCRIPTION
**Pull request by daryo918**
cart functionality

### CODE REVIEW

**Review by Nicolas000 — APPROVED** [review]

**Comment by esteban027** [comment]
Please change the objective branch to develop

File: app/src/controllers/products.controllers.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      message: "Error",
+      error: e
+    });
+  }
+};
+
+
+export const showCart=async (req: Request, resp: Response): Promise<Response> => {
+  return resp.status(200).json({
+    message: "mostrar productos"
```
Comment (esteban027) [inline app/src/controllers/products.controllers.ts]: Only english

File: app/src/controllers/products.controllers.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  });
+}
+export const deletePorductToCart=async (req: Request, resp: Response): Promise<Response> => {
+    return resp.status(200).json({
+        message: "quitar x producto"
```
Comment (esteban027) [inline app/src/controllers/products.controllers.ts]: only english please

File: app/src/controllers/products.controllers.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      });
+};
+
+export const payBill=async (req: Request, resp: Response): Promise<Response> => {
+    return resp.status(200).json({
+        message: "validar pago"
```
Comment (esteban027) [inline app/src/controllers/products.controllers.ts]: Only english please

File: postgres/users.sql [hunk 4]
```diff
@@ -1,49 +1,44 @@
...
-	CONSTRAINT fk_organizations
-      FOREIGN KEY(fk_organizations) 
-	 REFERENCES organization(id_organization)
-);
 
-CREATE TABLE organization (
-	id_organization serial PRIMARY KEY,
-	name_organization VARCHAR ( 255 )  NOT NULL,
-    status CHAR(1) 	NOT NULL
+create table producto(
```
Comment (esteban027) [inline postgres/users.sql]: only english please

File: postgres/users.sql [hunk 5]
```diff
@@ -1,49 +1,44 @@
...
+    nombre text,
+    precio float,
+    stock int
 );
 
-CREATE TABLE roles (
-	id_role serial PRIMARY KEY,
-	name_role VARCHAR ( 255 )  NOT NULL,
-    status CHAR(1) 	NOT NULL
+create table details(
```
Comment (esteban027) [inline postgres/users.sql:16]: only english please

File: postgres/users.sql [hunk 6]
```diff
@@ -1,49 +1,44 @@
...
 );
+
+alter table details add constraint fk_factura_detail foreign key (id_factura) references factura(id_factura);
+alter table details add constraint fk_producto_detail foreign key (id_producto) references producto(id_producto);
+alter table factura add constraint fk_factura_users foreign key (id_user) references users(id_user);
+
+insert into users(uuid, name, password,email) values ('f5086255-c946-4e86-84af-c1e8db4ab733','admin','$2a$10$5tzvcBq07JWWg/Baso8wou69Fhcd0crk/aatSP4TiRcU649CeH79a','joe1@pepito.com');
+
+
+insert into producto(nombre,precio,stock) values ('Coca-cola',1000,1000);
```
Comment (esteban027) [inline postgres/users.sql]: same comment as before

**Review by esteban027 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**